### PR TITLE
Use explicit parameter to control the bitorder presented in register_view

### PR DESCRIPTION
**Pull request by mandolaerik**
- Generate refman title page for github wiki
- Use explicit parameter to control the bitorder presented in register_view

### CODE REVIEW

**Comment by syssimics** [comment]
TeamCity build completed (SUCCESS): https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=9566648

**Comment by syssimics** [comment]
TeamCity build completed (FAILURE): https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=9566647

**Comment by syssimics** [comment]
TeamCity build completed (SUCCESS): https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=9566675

**Comment by syssimics** [comment]
TeamCity build completed (SUCCESS): https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=9566674

**Comment by syssimics** [comment]
TeamCity build completed (FAILURE): https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=9568715

**Comment by syssimics** [comment]
TeamCity build completed (FAILURE): https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=9568714

**Comment by syssimics** [comment]
TeamCity build completed (FAILURE): https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=9573510

**Comment by syssimics** [comment]
TeamCity build completed (FAILURE): https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=9573509

**Comment by syssimics** [comment]
TeamCity build completed (FAILURE): https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=9577063

**Comment by syssimics** [comment]
TeamCity build completed (FAILURE): https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=9577062

**Comment by syssimics** [comment]
TeamCity build completed (SUCCESS): https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=9577139

File: lib/1.4/dml-builtins.dml [hunk 1]
```diff
@@ -1976,21 +1995,7 @@ template bank is (object, shown_desc, _qname) {
         // as it stands, DML prevents us from deducing the
         // bitorder directly so we rely on the XML 'guessing'
```
Comment (lwaern-intel) [inline lib/1.4/dml-builtins.dml:1996]: Is this comment still relevant?

File: lib/1.4/dml-builtins.dml [hunk 2]
```diff
@@ -1976,21 +1995,7 @@ template bank is (object, shown_desc, _qname) {
...
-            local attr_value_t ret;
-            ret = VT_call_python_module_function("register_view",
-                                                 "big_endian_bitorder",
-                                                 &args);
-            SIM_attr_free(&args);
-
-            local bool big_endian = SIM_attr_boolean(ret);
-            SIM_attr_free(&ret);
-            return big_endian;
+            return bank.be_bitorder;
```
Comment (lwaern-intel) [inline lib/1.4/dml-builtins.dml:1998]: Could the changes to `big_endian_bitorder()` cause any potential breakage?

**Comment by syssimics** [comment]
TeamCity build completed (SUCCESS): https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=9577138

File: lib/1.4/dml-builtins.dml (hunk 2 above)
Comment (mandolaerik) [reply lib/1.4/dml-builtins.dml:1998]: No, for two reasons:
1. big_endian_bitorder only affects presentation, not simulation. When it returns true, some UIs will display bit numbers in the wrong order.
2. The autoparam is set to the same value as was written in the XML entry

File: lib/1.4/dml-builtins.dml (hunk 1 above)
Comment (mandolaerik) [reply lib/1.4/dml-builtins.dml:1996]: well spotted, will remove

**Comment by mandolaerik** [comment]
Fixed Love's comment and added a releasenote.

**Comment by syssimics** [comment]
TeamCity build completed (SUCCESS): https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=9577538

**Review by svenwest — APPROVED** [review]
The code in the PR looks good. No objections from me.

> Generate refman title page for github wiki

This line in the comment/description for the PR looks odd.

**Comment by syssimics** [comment]
TeamCity build completed (SUCCESS): https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=9577537

**Pull request merged**